nise/yaml_generators/oci/generator.py

```diff
@@ -68,7 +58,7 @@ def generate_tags(config, key):
 
     tags = []
     if not config.get("tags"):
-        keys = TAG_KEYS.get(key)
+        keys = get_tag_keys(key)
```

Comment:
```suggestion
        keys = oci_tags.get(key)
```

Reply:
Alternatively, just use `getattr` directly.
```suggestion
        keys = getattr(oci_tags, key, None)
```